generic/GenericCplComp.F90

```diff
@@ -1338,7 +1340,7 @@ subroutine ReadRestart(CC, SRC, DST, CLOCK, RC)
           case default
              _ASSERT(.false., "Unsupported rank")
           end select
-          if(associated(mask)) deallocate(mask)
+          DEALOC_(mask)
```

Comment:
I also suggest renaming the macro to have a leading underscore `_DEALOC` unless that is too different than existing practice.

Reply:
_DEALLOC works